tox.ini

```diff
@@ -5,7 +5,7 @@
 [tox]
 envlist =
 # Basic configurations: Run the tests for each python version.
-    py{39, 310, 311}-dev
```

Comment:
I think this needs to be -prod otherwise the osml-imagery-toolkit dependency on line 24 will not come in.

Reply:
Fixed and tested